ml-agents/mlagents/trainers/torch/utils.py

```diff
@@ -158,18 +174,38 @@ def create_input_processors(
         :param unnormalized_inputs: Vector inputs that should not be normalized, and added to the vector
             obs.
         :param normalize: Normalize all vector inputs.
-        :return: Tuple of visual encoders and vector encoders each as a list.
+        :return: Tuple of :
+         - ModuleList of the encoders (None if the input requires to be processed with a variable length
+         observation encoder)
+         - A list of embedding sizes (0 if the input requires to be processed with a variable length
+         observation encoder)
+         - A list of the inputs that need to be processed by a variable length observation encoder.
```

Comment:
Is this comment accurate? It makes it sound like you should be returning (ModuleList, list[int], list[int])

Reply:
You are right, I fixed it in the latest commit.